KeeTrayTOTP/FormAbout.cs

```diff
@@ -156,5 +159,46 @@ private void FormAbout_FormClosed(object sender, FormClosedEventArgs e)
         {
             GlobalWindowManager.RemoveWindow(this);
         }
+
+        /// <summary>
+        /// Windows Form Mouse Click Handler.
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void ListViewAbout_MouseClick(object sender, MouseEventArgs e)
+        {
+            var hit = ListViewAbout.HitTest(e.Location);
+            if (hit.Item != null && hit.Item == ListViewAbout.Items[4] && hit.SubItem == ListViewAbout.Items[4].SubItems[1])
+            {
+                var url = new Uri(hit.SubItem.Text); 
+                System.Diagnostics.Process.Start(url.ToString());
+            }
+        }
+        /// <summary>
+        /// Windows Form Row Subitem To Underlined Font.
+        /// </summary>
+        /// <param name="listView"></param>
+        /// <param name="RowIndex"></param>
+        /// /// <param name="color"></param>
+        private void ChangeToUnderlined(ref ListView listView, int RowIndex, Color color)
```

Comment:
IMO the method name isn't ideal. It's not only changing the listViewSubItem to "Underlined", it also sets the color.
Maybe something like: "SetHyperlinkStyle"

The ``ref`` keyword is not needed, as ``ListView`` is a reference type by it self (see [Reference types - C# Reference | Microsoft Docs](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/reference-types)).

You can even omit the ``listView`` parameter and just access the ``ListViewAbout`` member of the class.